.chglog/config-release.yml

```diff
@@ -0,0 +1,36 @@
+style: github
+template: RELEASE.tpl.md
+info:
+  title: RELEASE
+  repository_url: https://github.com/sysdiglabs/charts
+options:
+  commits:
+    filters:
+      Type:
+        - feat
+        - new
+        - fix
+        - perf
+        - refactor
+        - chore
+        - docs
+        - github_actions_ci
```

Comment:
I am assuming from here on the commit messages would have to include these filters as prefixes. This will require telling every person who commits to have that tag present correct?

Reply:
If the commit does not match the regex pattern it will be listed in the "Others" section and will not have as much details
So yes ideally every commit should adhere to these standards